src/app/public/modules/description-list/description-list.component.html

```diff
@@ -0,0 +1,33 @@
+<dl
+  class="sky-description-list"
+  [ngClass]="{
+      'sky-description-list-long-description-mode' : mode == 'longDescription',
+      'sky-description-list-default-mode' : mode == 'standard',
+      'sky-description-list-horizontal' : orientation == 'horizontal'
+    }"
+  #descriptionListElement
+>
+  <!-- Standard mode needs a wrapping div to allow horizontal flexing. -->
+  <ng-container>
```

Comment:
Is there a way to reduce the complexity of the template and let the pieces "fall where they lay"? I'm confused why we need to use `ng-container` and `ng-template`? Thanks for explaining it to me!

Reply:
Perhaps. A lot of this was to enforce the wrapping of stuff with the appropriate `<dt>` `<dd>` tags. Now that we're not coupled to the old component, I might be able to clean this up. I'll take a look....

Reply:
Glad you brought this up - I do not need this logic anymore now that we've decoupled. I've simplified it greatly and added comments on what's left. Does this work better?